Cargo.toml

```diff
@@ -95,3 +95,4 @@ tracing = "0.1"
 tryhard = "0.5.1"
 which = "4.4.0"
 wiremock = "0.5"
+tokio-stream = { version = "0.1.14", features = ["net"] }
```

Comment:
Don't activate features in the workspace crate - activate them in the crates that make use of the workspace dependency.